lib/Comment.php

```diff
@@ -46,6 +46,11 @@ class Comment extends Core implements CoreInterface, MetaInterface {
 	 */
 	public $id;
 
+	/**
+	 * @var int
```

Comment:
I think in the future we should try to get properties fully documented, including summary and descriptions, because these will also appear in the documentation. But it’s totally fine for the scope of this pull request.

Reply:
Agreed. There are LOTS of them so good for a future PR/task